groups/bdl/bdlb/bdlb_guid.h

```diff
@@ -517,6 +521,13 @@ unsigned char Guid::version() const {
     return uc(timeHiAndVersion() >> 12);
 }
 
+template <class HASH_ALGORITHM>
+void hashAppend(HASH_ALGORITHM& hashAlg, const Guid& guid)
```

Comment:
This method should be declared above.  Additionally the definition is usually placed outside of the package namespace, and thus qualified with the package namespace (I'm not sure why).